### PR TITLE
Simon/non linear budget intervals

### PR DESCRIPTION
**Pull request by SimonChow9527**
[AB#91](https://dev.azure.com/Tringo/cabcbfb6-3a22-44cf-ac98-b3810925a870/_workitems/edit/91)

### CODE REVIEW

File: app/src/components/searchWidget/budgetRangeSlider.tsx [hunk 1]
```diff
@@ -42,8 +43,20 @@ class BudgetRangeSlider extends React.Component<SliderProps, SliderState> {
         this.props.onChange && this.props.onChange([this.props.min, sliderValue]);
     }
 
+    getStep(){
+
```
Comment (rmit-s3668804-Dzmitry-Kakaruk) [inline app/src/components/searchWidget/budgetRangeSlider.tsx]: Fix formatting, remove empty line

File: app/src/components/searchWidget/budgetRangeSlider.tsx [hunk 2]
```diff
@@ -42,8 +43,20 @@ class BudgetRangeSlider extends React.Component<SliderProps, SliderState> {
...
+        if(this.state.currentValue>=Math.round(this.props.max*
+            (process.env.REACT_APP_SLIDER_HIGHEST_TRESHOLD?parseFloat(process.env.REACT_APP_SLIDER_HIGHEST_TRESHOLD):0.67)))
+        return process.env.REACT_APP_SLIDER_HIGHEST_STEP?parseInt(process.env.REACT_APP_SLIDER_HIGHEST_STEP):this.props.step;      
+        else if(this.state.currentValue>=Math.round(this.props.max*
+            (process.env.REACT_APP_SLIDER_MEDIUM_TRESHOLD?parseFloat(process.env.REACT_APP_SLIDER_MEDIUM_TRESHOLD):0.34)))
+        return process.env.REACT_APP_SLIDER_MEDIUM_STEP?parseInt(process.env.REACT_APP_SLIDER_MEDIUM_STEP):this.props.step;
+        else
+        return process.env.REACT_APP_SLIDER_LOWEST_STEP?parseInt(process.env.REACT_APP_SLIDER_LOWEST_STEP):this.props.step;
+    }
+
```
Comment (rmit-s3668804-Dzmitry-Kakaruk) [inline app/src/components/searchWidget/budgetRangeSlider.tsx:75]: Also, did you disable ESLint or never enabled 'cause still using bs code? Our rules do not allow for non-curly one line if statements

**Review by rmit-s3668804-Dzmitry-Kakaruk — DISMISSED** [review]

**Comment by SimonChow9527** [comment]
formatting done

**Review by s3419069 — APPROVED** [review]
now it looks good

**Pull request merged**